theano/gpuarray/dnn.py

```diff
@@ -987,10 +951,10 @@ def dnn_conv(img, kerns, border_mode='valid', subsample=(1, 1),
                    shape_i(img, 2, fgraph) - shape_i(kerns, 2, fgraph) + 1,
                    shape_i(img, 3, fgraph) - shape_i(kerns, 3, fgraph) + 1)
         out_shp = assert_conv_shape(out_shp)
-        out = gpu_alloc_empty(ctx_name, dtype=img.dtype)(*out_shp)
+        out = GpuAllocEmpty(ctx_name, dtype=img.dtype)(*out_shp)
```

Comment:
swap the 2 param. Make sure this is also done for all instance of GpuAllocEmpty introduced. gpu_alloc_empty and GpuAllocEmpty don't take the parameter in the same order.

Or better, pass both param by name. This will always work.